crates/matrix-sdk-common/src/deserialized_responses.rs

```diff
@@ -305,10 +305,14 @@ pub struct EncryptionInfo {
 #[derive(Clone, Deserialize, Serialize)]
 pub struct SyncTimelineEvent {
     /// The actual event.
-    pub event: Raw<AnySyncTimelineEvent>,
+    #[serde(rename = "event")]
+    inner_event: Raw<AnySyncTimelineEvent>,
+
     /// The encryption info about the event. Will be `None` if the event was not
     /// encrypted.
-    pub encryption_info: Option<EncryptionInfo>,
+    #[serde(rename = "encryption_info")]
+    inner_encryption_info: Option<EncryptionInfo>,
```

Comment:
Ditto, not sure about the `inner_` prefix here, could we remove it?

Reply:
As above: yes, `encryption_info` would have been a better name, but it's going away again anyway.